tor/onionaddr.go

```diff
@@ -45,6 +45,9 @@ type OnionAddr struct {
 
 	// Port is the port of the onion address.
 	Port int
+
+	// PrivateKey is the onion address' private key. Only returned if requested.
```

Comment:
nit: this comment is no longer accurate, we're now always returning the private key.